nipype/pipeline/engine/nodes.py

```diff
@@ -189,12 +188,11 @@ def interface(self):
 
     @property
     def result(self):
-        if self._result:
-            return self._result
```

Comment:
i think this was a legacy from before. we should check when `self._result` is actually not None.